Modules/HR/Observers/Recruitment/ApplicantObserver.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Modules\HR\Observers\Recruitment;
+
+use Modules\HR\Entities\Applicant;
+use Modules\HR\Entities\University;
+use Modules\HR\Entities\UniversityAlias;
+
+class ApplicantObserver
+{
+    /**
+     * Listen to the Applicant create event.
+     *
+     * @param  \Modules\HR\Entities\Applicant  $Applicant
+     * @return void
+     */
+    public function created(Applicant $applicant)
+    {
+        $data = request()->all();
+
+        if (isset($data['college'])) {
+            $university = University::select('id')->where('name', $data['college'])->first();
+            if ($university == null) {
```

Comment:
You need to make your code short. You can write it like this:

```suggestion
            if (!$university) {
```